packages/nextjs/components/ExampleUi/ContractInteraction/CopyIcon.tsx

```diff
@@ -0,0 +1,28 @@
+type CopyIconProps = {
```

Comment:
Maybe we can move this icons into a `icons` or `assets` folder? Just for the sake of having a cleaner folder content.

Not sure if right below `ExampleUi` or `ExampleUi/ContractInteraction`.